src/test/resources/alma/HT012734884.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Not sure why adding this file. What does it test what other files not already test?

Reply:
Thought I should only, or all files that change concerning the change in the morph. Will reduce the number of testfiles in the future.

Reply:
I don't understand what you mean by this.
If this resource doesn't test anything new it should not be commited.

Reply:
+1

Reply:
Discussed offline: this resource checks multiple inCollection eResources, so this is ok.